cfy_manager/utils/service.py

```diff
@@ -292,11 +292,14 @@ def using_systemd_service(service_name):
         # no systemctl at all, so there for sure won't be systemd services
         return False
 
-    if subprocess.run(
+    is_enabled = subprocess.run(
         ['/bin/systemctl', 'is-enabled', service_name],
         stdout=subprocess.PIPE,
         stderr=subprocess.DEVNULL,
-    ).stdout.strip().lower() == 'enabled':
+    ).stdout.strip().lower()
+    if isinstance(is_enabled, bytes):
```

Comment:
no need, this will always be bytes, cf [docs](https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.stdout)